src/gobject.cc

```diff
@@ -427,18 +427,20 @@ static MaybeLocal<FunctionTemplate> NewClassTemplate (GIBaseInfo *info, GType gt
 
     const char *class_name = g_type_name (gtype);
 
-    auto tpl = New<FunctionTemplate> (GObjectConstructor, New<External> (info));
+    auto tpl = New<FunctionTemplate> (GObjectConstructor, New<External>((void *) gtype));
     tpl->SetClassName (UTF8(class_name));
     tpl->InstanceTemplate()->SetInternalFieldCount(1);
 
-    GIObjectInfo *parent_info = g_object_info_get_parent (info);
-    if (parent_info) {
-        auto parent_tpl = GetClassTemplateFromGI ((GIBaseInfo *) parent_info);
+    GType parent_type = g_type_parent(gtype);
+    if (parent_type == 0) {
```

Comment:
There is probably some constant (`G_TYPE_NONE` maybe?) to use here instead of 0. Zero doesn't carry any semantic value, as opposed to a named constant.